modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/checkpoint/Checkpointer.java

```diff
@@ -496,6 +496,9 @@ private void doCheckpoint() {
             }
 
             updateMetrics(chp, tracker);
+
+            if (cacheProcessor.context().kernalContext().performanceStatistics().enabled())
```

Comment:
Can we cache the value of `PerformanceStatisticProcessor` in the instance variable?

Reply:
when initializing the Checkpointer 
cacheProcessor.context().kernalContext().performanceStatistics is null